src/memfs.ts

```diff
@@ -233,6 +232,11 @@ export class InMemoryFileSystem implements ts.ParseConfigHost, ts.ModuleResoluti
 	trace(message: string) {
 		this.logger.log(message);
 	}
+
+	// TODO: move this method to util and use through all the code
+	private path2uri(path: string): string {
```

Comment:
I see it resolves relative paths, in what cases is it called with a relative path?